### PR TITLE
Flask: capture responseHeadersOnEntrySpans

### PR DESCRIPTION
**Pull request by GSVarsha**
Flask: capture responseHeadersOnEntrySpans

Signed-off-by: Varsha GS [varsha.gs@ibm.com](mailto:varsha.gs@ibm.com)

### CODE REVIEW

File: instana/instrumentation/flask/vanilla.py [hunk 1]
```diff
@@ -17,6 +17,21 @@
 path_tpl_re = re.compile('<.*>')
 
 
+def extract_custom_headers(span, headers, format):
```
Comment (Ferenc-) [inline instana/instrumentation/flask/vanilla.py:20]: If the extract function is the same for both vanilla and with_blinker, then we could put this function into a separate file and include it here and into with_blinker.

Comment (GSVarsha) [reply instana/instrumentation/flask/vanilla.py:20]: @Ferenc- Is it okay if I add the function in [this](https://github.com/instana/python-sensor/blob/master/instana/instrumentation/flask/common.py) file?

Comment (Ferenc-) [reply instana/instrumentation/flask/vanilla.py:20]: I think it could fit there, yes.

**Review by Ferenc- — APPROVED** [review]
LGTM

**Review by pvital — APPROVED** [review]
It looks good to me.

**Pull request merged**